client/app/admin/admin.component.html

```diff
@@ -0,0 +1,148 @@
+<div class="container-fluid fill-height p-0">
+
+  <div class="row half">
+    <div id="menuBar" class="col-3 sidebar pl-0">
+      <button class="savebtn" (click)="saveMenu()">Save</button>
+      <button class="closebtn" (click)="closeMenuNav()">Edit Menu -</button>
```

Comment:
do you need the "-" in 'Edit Menu -'?

Reply:
Sorry. just noticed that lower down you have 'Edit Menu +', so i guess it makes sense when you see the UI, not just the code. :)